conf/resources/data/etmp.eligibility.SIMP/hasInvalidInterestSignalsTrue.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
can this be renamed to "rlsSignalValueTrue.json as it looks like its validating "RLS"